front/src/modules/simulationResult/components/SpaceTimeChartWrapper/useTrackOccupancy.ts

```diff
@@ -162,35 +194,90 @@ const useTrackOccupancy = ({
     []
   );
 
+  const toOwnerTimetableItemId = (id: TrainId): TimetableItemId => {
+    if (isTrainScheduleId(id) || isPacedTrainId(id)) return id;
+    return extractPacedTrainIdFromOccurrenceId(id);
+  };
+
   const fetchTrackOccupancy = useCallback(
     async (
       opId: string | undefined | null,
       trainsCollection: Record<TimetableItemId, TrainSpaceTimeData>
-    ): Promise<MovableOccupancyZone[]> =>
-      opId
-        ? flatMap(
-            (
-              await postTrainScheduleTrackOccupancy({
-                body: {
-                  operational_point_id: opId,
-                  infra_id: infraId,
-                  train_schedule_ids: Object.keys(trainsCollection)
-                    .filter((id) => isTrainScheduleId(id))
-                    .map((id) => extractEditoastIdFromTrainScheduleId(id)),
-                },
-              })
-            ).data,
-            (entries, trackId) =>
-              entries.map((entry) =>
-                getMovableOccupancyZone(
-                  trackId,
-                  entry,
-                  trainsCollection[formatEditoastIdToTrainScheduleId(entry.train_schedule_id)]
-                )
-              )
-          ) // TODO : append result of postPacedTrainTrackOccupancy
-        : [],
-    [infraId]
+    ): Promise<MovableOccupancyZone[]> => {
+      if (!opId) return [];
+
+      const scheduleIds: TrainScheduleId[] = [];
+      const pacedIds: PacedTrainId[] = [];
```

Comment:
`schedule` is also the schedule of a train (`trainSchedule` or `pacedTrain`) and `paced` is used to name the special props of a paced train (`interval` and `time_window`) ;) 
```suggestion
      const trainScheduleIds: TrainScheduleId[] = [];
      const pacedTrainIds: PacedTrainId[] = [];
```